cli/context/store/storeconfig.go

```diff
@@ -11,15 +11,15 @@ type NamedTypeGetter struct {
 	typeGetter TypeGetter
 }
 
-// EndpointTypeGetter returns a NamedTypeGetter with the spcecified name and getter
+// EndpointTypeGetter returns a NamedTypeGetter with the specified name and getter
 func EndpointTypeGetter(name string, getter TypeGetter) NamedTypeGetter {
 	return NamedTypeGetter{
 		name:       name,
 		typeGetter: getter,
 	}
 }
 
-// Config is used to configure the metadata marshaler of the context store
+// Config is used to configure the metadata marshaller of the context store
```

Comment:
I think both are correct https://en.wiktionary.org/wiki/marshaler, but Golang uses single `l` in its vocabulary, e.g. https://golang.org/pkg/encoding/#TextMarshaler

Reply:
my IDE warned me about a (supposed to be) typo and proposed to fix it.
Didn't know about those distinct-but-valid forms.